### PR TITLE
Worked on sleep statistics by adding an interface for users to input …

### PR DESCRIPTION
**Pull request by joshuamkchoi1103**
Users can input their sleep hours by entering sleep time to wake up time and it will calculate the amount of sleep for the day and for the week. Put this under statistics.

### CODE REVIEW

**Comment by htluu-ucsd** [comment]
Renamed a component and commented out 1 test on App.test.tsx

File: client/src/App.test.tsx [hunk 1]
```diff
@@ -3,10 +3,8 @@ import {render, screen} from '@testing-library/react';
...
-test('test renders main page', () => {
-    render(
-        <BrowserRouter>
-            <App/>
-        </BrowserRouter>
-    );
+test('renders learn react link', () => {
+  // render(<App />);
+  // const linkElement = screen.getByText(/learn react/i);
+  // expect(linkElement).toBeInTheDocument();
```
Comment (Shuzhengz) [inline client/src/App.test.tsx:9]: a bit confused about this change

**Review by Shuzhengz — CHANGES_REQUESTED** [review]
also add newlines to the end of files, lookout for these warnings:

image

File: client/src/App.test.tsx (hunk 1 above)
Comment (joshuamkchoi1103) [reply client/src/App.test.tsx:9]: @htluu-ucsd

Comment (htluu-ucsd) [reply client/src/App.test.tsx:9]: i just wanted to run my own tests, can just be reverted

Comment (Shuzhengz) [reply client/src/App.test.tsx:9]: it seems like it's the code from `start-react-app` so it probably got mixed into it somehow. can you revert it? thx!

**Review by htluu-ucsd — APPROVED** [review]

**Review by Shuzhengz — APPROVED** [review]
might wanna fix those newline warnings, but otherwise lgtm

**Pull request merged**